src/qibocal/fitting/classifier/run.py

```diff
@@ -47,6 +32,14 @@ class Classifiers(enum.Enum):
 base_dir = pathlib.Path()
 
 
+def import_classifiers(cls_names: list[str]):
+    importing_func = lambda mod: importlib.import_module(
+        ".." + mod, "qibocal.fitting.classifier.*"
+    )
+    Classifiers = list(map(importing_func, cls_names))
+    return Classifiers
```

Comment:
```suggestion
    classifiers = list(map(importing_func, cls_names))
    return classifiers
```

Only classes should be named in Pascal case

Reply:
However, many kudos for the `list(map(...))` instead of the list comprehension `[importing_func(c) for c in cls_names]`.

I like both of them (both directly coming from FP), but I had the feeling no one would have ever used the first one xD